third_party/ijar/test/ijar_test.sh

```diff
@@ -41,7 +41,7 @@ shift
 source ${DIR}/testenv.sh || { echo "testenv.sh not found!" >&2; exit 1; }
 
 function cleanup() {
-  rm -fr "$TEST_TMPDIR"/*
+  rm -fr "${TEST_TMPDIR:?}"/*
```

Comment:
replace ? with ___do__not__exists___ or something like that.

Reply:
How about `"${TEST_TMPDIR:-sentinel}"` ?